README.md

```diff
@@ -249,6 +249,8 @@ TSC members can nominate new members at any time. Candidates for membership tend
 | ![](http://oi65.tinypic.com/2vif56q.jpg) <br> [cmdc0de](https://github.com/cmdc0de) - Demetrius Comes (GoDaddy) | ![](https://avatars3.githubusercontent.com/u/20224954?s=100&v=4) <br> [gdams](https://github.com/gdams) - George Adams (Microsoft) | ![](https://avatars3.githubusercontent.com/u/357222?s=100&v=4) <br> [jerboaa](https://github.com/jerboaa) - Severin Gehwolf (Red Hat) | 
 | ![](https://avatars0.githubusercontent.com/u/1615532?s=100&v=4) <br> [johnoliver](https://github.com/johnoliver) - John Oliver (Microsoft) | ![](https://avatars1.githubusercontent.com/u/180840?s=100&v=4) <br> [karianna](https://github.com/karianna) - Martijn Verburg (Microsoft) | ![](https://avatars3.githubusercontent.com/u/2836948?s=100&v=4) <br> [smlambert](https://github.com/smlambert) - Shelley Lambert (IBM) |
 | ![](https://avatars0.githubusercontent.com/u/6487691?s=100&v=4) <br> [sxa555](https://github.com/sxa555) - Stewart X Addison (IBM) | ![](https://avatars1.githubusercontent.com/u/621161?s=100&v=4) <br> [tellison](https://github.com/tellison) - Tim Ellison (IBM) | ![](https://avatars2.githubusercontent.com/u/35047?s=100&v=4) <br> [vielmetti](https://github.com/vielmetti) - Edward Vielmetti (Packet)
+| ![](https://avatars0.githubusercontent.com/u/9443847?s=100&v=4) <br> [hendrikebbers](https://github.com/hendrikebbers) - Hendrik Ebbers (Karakun AG) ||
+
 
```

Comment:
Should likely remove the '(listed alphabetically)' bit from above or re-order.  Preference to remove it, as it may continue to catch us as TSC members change/update.

Reply:
@hendrikebbers - can you make the change as suggested?